lib/reply.js

```diff
@@ -47,20 +50,20 @@ Reply.prototype.send = function (payload) {
     return
   }
 
-  const contentType = this.res.getHeader('Content-Type')
-  const contentTypeNotSet = contentType === undefined
+  var contentType = getHeader(this, 'content-type')
+  var contentTypeNotSet = contentType === undefined
```

Comment:
Is the change to `var` here an optimization?

Reply:
It might be, but I didn't measure without.

Reply:
Maybe change it back for consistency?

Reply:
@jsumners as far as I know `var` is slightly faster than `const` because of the scoping.

Reply:
@jsumners @delvedor I tried reverting the const -> var change and the context thing, and I got a slight downgrade of the throughput. I prefer to leave them as they are in this PR.

Reply:
Which means the answer to the question is "yes".

Reply:
yes.